distribution/server-x-dist/assembly.xml

```diff
@@ -31,10 +31,6 @@
             <outputDirectory/>
```

Comment:
I'm not quite following all these changes - are all this related to themes changes?

Reply:
Not related, but removing unnecessary code. Going to send a separate PR.